apps/sasquatch/build.gradle

```diff
@@ -81,10 +81,9 @@ repositories {
 }
 
 dependencies {
-    implementation "com.android.support:support-v4:${rootProject.ext.supportLibVersion}"
-    implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
-    implementation "com.android.support:customtabs:${rootProject.ext.supportLibVersion}"
-    implementation "com.android.support:recyclerview-v7:${rootProject.ext.supportLibVersion}"
+    implementation "androidx.appcompat:appcompat:1.0.2"
+    implementation "androidx.browser:browser:1.0.0"
+    implementation "androidx.recyclerview:recyclerview:1.1.0"
```

Comment:
Does it mean that all applications that use SDK have to change it? Is it breaking change?

Reply:
Nope, these changes need only for the correct passing of our CI steps